test/quantization/core/test_quantized_module.py

```diff
@@ -155,15 +154,16 @@ def _test_linear_api_impl(self, batch_size, in_features, out_features, use_bias,
                          linear_unpack(loaded_qlinear._packed_params._packed_params))
         self.assertEqual(qlinear.scale, loaded_qlinear.scale)
         self.assertEqual(qlinear.zero_point, loaded_qlinear.zero_point)
-        # make sure loaded_qlinear has the same dir as qlinear since
-        # scripting the module will add __overloads__ to __dict__
-        self.checkScriptable(loaded_qlinear, [[X_q]], check_save_load=True)
+        # scripting will add __overloads__ to __dict__, which is why we script a copy
+        # to be able to do the check in the next line
+        self.checkScriptable(copy.deepcopy(loaded_qlinear), [[X_q]], check_save_load=True)
```

Comment:
nit: should probably do this in `checkScriptable` itself so that it works for other cases as well

Reply:
sure, can do this in a future PR